db/migrate/20170824170659_add_anything_else_to_snap_application.rb

```diff
@@ -0,0 +1,5 @@
+class AddAnythingElseToSnapApplication < ActiveRecord::Migration[5.1]
+  def change
+    add_column :snap_applications, :anything_else, :text
```

Comment:
on GCF they call this field `additional_information` -- thoughts on that name?

Reply:
My thoughts are that it's a better name :) 2f30308